src/components/icons/index.ts

```diff
@@ -79,4 +82,13 @@ export const TransactionInsightsIcon = wrapIcon(
   defineMessage`Security`,
 );
 
+export const AuditorIcon = wrapIcon(BaseAuditorIcon, defineMessage`Auditor`);
+
+export const ReviewerIcon = wrapIcon(BaseReviewerIcon, defineMessage`Reviewer`);
+
+export const DeveloperIcon = wrapIcon(
+  BaseDeveloperIcon,
+  defineMessage`Developer`,
+);
```

Comment:
The icons in this file are sorted alphabetically.

Reply:
noted

Reply:
updated